src/sas/qtgui/Perspectives/Fitting/FittingWidget.py

```diff
@@ -1075,6 +1078,8 @@ def editConstraint(self):
         constraint.validate = mc_widget.validate
 
         # Which row is the constrained parameter in?
+        if model_key == 'poly' and 'Distribution' in constraint.param:
```

Comment:
Shouldn't "Distribution" be enough? All PD parameter names start with "Distribution" and I'm not sure whether any have "poly" in them